test/Window/Keyboard.test.cpp

```diff
@@ -0,0 +1,61 @@
+#include <SFML/Window/Keyboard.hpp>
+
+// Other 1st party headers
+#include <SFML/System/String.hpp>
+
+#include <catch2/catch_test_macros.hpp>
+
+#include <WindowUtil.hpp>
+
+// We're limited on what can be tested. For example the description for certain
+// scancodes is language-specific and the mapping between keys and scancodes is
+// hardware-specific. Without control over the hardware and the configuration of
```

Comment:
To be more specific, descriptions depend not only on language but also on OS.
I would say that mapping between keys and scancodes is more generally a user configuration rather than a hardware setting. Users probably choose a configuration that matches their hardware but not necessarily.